# A representation of position and velocity of object
Header header
float32 pos_x
float32 pos_y
float32 pos_z
float32 vel_x
float32 vel_y
float32 vel_z
int32 class
